packages/mui-base/src/AutocompleteUnstyled/useAutocomplete.js

```diff
@@ -527,16 +527,27 @@ export default function useAutocomplete(props) {
     // eslint-disable-next-line react-hooks/rules-of-hooks
     React.useEffect(() => {
       if (!inputRef.current || inputRef.current.nodeName !== 'INPUT') {
-        console.error(
-          [
-            `MUI: Unable to find the input element. It was resolved to ${inputRef.current} while an HTMLInputElement was expected.`,
-            `Instead, ${componentName} expects an input element.`,
-            '',
-            componentName === 'useAutocomplete'
-              ? 'Make sure you have binded getInputProps correctly and that the normal ref/effect resolutions order is guaranteed.'
-              : 'Make sure you have customized the input component correctly.',
-          ].join('\n'),
-        );
+        if (inputRef.current && inputRef.current.nodeName === 'TEXTAREA') {
+          console.warn(
+            [
+              `A textarea element was provided to ${componentName} where input was expected.`,
+              `This is not a supported scenario but it may work under certain conditions.`,
+              `A textarea keyboard navigation may conflict with Autocomplete controls (e.g. enter and arrow keys).`,
+              `Make sure to test keyboard navigation and add custom event handlers if necessary.`,
+            ].join('\n'),
+          );
```

Comment:
IMHO this is wrong. We can't log a warning if it can work. I think that it's either:

```suggestion
          console.error(
            [
              `A textarea element was provided to ${componentName} where input was expected.`,
              `This is not a supported scenario.`,
              `A textarea keyboard navigation conflicts with the autocomplete controls (e.g. enter and arrow keys).`,
            ].join('\n'),
          );
```

or we 

```suggestion
```

Reply:
Hello, is there any way to disable this warning? A textarea works fine for what I need and this warning shows up all the time, filling the console with unnecessary logs. Please let me know @oliviertassinari , maybe you can help?